ui/app/components/admin-page/admin-page.html

```diff
@@ -33,6 +33,11 @@ <h4 class="admin-page__group-list-subheader">Edit {{ group.name }}</h4>
           <div class="admin-page__input-help-text">i.e. In this group, members of {{ group.name }} fund projects to improve their work environment and support each other.</div>
           <a class="admin-page__description-markdown-link" href="https://www.loomio.org/markdown" target="_blank">formatting help</a>
         </md-input-container>
+        <md-input-container class="admin-page__input md-block">
+          <label>Add Funds</label>
+          <input ng-model="group.addFunds" type="url"/>
+          <div class="admin-page__input-help-text">A url of a page with info about adding funds to an account (include http(s)://).</div>
```

Comment:
is this going to be a url or MD block which can have a link in it?
Will try out the staging server and see how this behaves.

I think the `include http(s)` is probably going to confuse people. If this is only ever going to be a link then maybe we shouldn't use MD, and instead should just turn this into an `<a>` and validate the input is a link with `http(s)://www` ..

Reply:
I think the https thing is going to confuse some people, might need so validation when they get it wrong / right?